packages/sdk/wallets/wallet-keystore/src/keystore-wallet-wrapper.ts

```diff
@@ -0,0 +1,42 @@
+import { LocalWallet } from '@celo/wallet-local'
+import { KeystoreBase } from './keystores'
+
+/**
+ * Convenience wrapper of the LocalWallet to connect to a keystore
+ */
+export class KeystoreWalletWrapper {
+  private _keystore: KeystoreBase
+  private _localWallet: LocalWallet
+
+  constructor(keystore: KeystoreBase) {
+    this._keystore = keystore
+    this._localWallet = new LocalWallet()
+  }
+
+  async importPrivateKey(privateKey: string, passphrase: string) {
+    await this._keystore.importPrivateKey(privateKey, passphrase)
+    this._localWallet.addAccount(privateKey)
+  }
+
+  getLocalWallet(): LocalWallet {
+    return this._localWallet
+  }
+
+  getKeystore(): KeystoreBase {
+    return this._keystore
+  }
+
+  async unlockAccount(address: string, passphrase: string) {
+    // Unlock and add account to internal LocalWallet
+
+    // TODO duration...seems non-trivial; as a default make this manual?
```

Comment:
I don't see how we implement duration in any meaningful way at this level.

The benefit of services/daemons is that they can clean themselves up, in JS/TS land there's no 100% sure way to know when the process will exit I believe.

Reply:
Yeah I think duration is going to be a bit weird; currently, I think it may actually work at the wrapper level though, since the `Keystore` isn't really holding anything in memory -- it has to decrypt the keystore file every time. Since the `LocalWallet` is where the signing fun happens, as long as the wrapper is still running, we could impose a time duration on when to remove the PK signer, but if the process gets killed, there isn't a security risk since the keystore files remain encrypted and not accessible. What do you think? definitely something that I'd be down to discuss/talk through (+ maybe even punt + revisit esp. once it becomes clear how this will be typically used)!

Reply:
Do we even need this feature at all? I.e. it makes sense with a separate process signer like geth/clef, but I imagine the use cases for this to be very different and I would probably just not have this feature?

Reply:
agreed; will remove this TODO!